libraries/aws_kms_key.rb

```diff
@@ -39,6 +39,24 @@ def exists?
     !@key.empty?
   end
 
+  def kms_tags(tag_list)
+    return {} if tag_list.nil? || tag_list.empty?
+    tags = {}
+    tag_list.each do |tag|
+      tags[tag[:tag_key]] = tag[:tag_value]
+    end
+    tags
+  end
+
+  def tags
+    begin
+      tag_list = @aws.kms_client.list_resource_tags(key_id: @display_name).tags
+    rescue
+      return {}
+    end
+    kms_tags(tag_list)
+  end
```

Comment:
actually, I tried the below, I was able to query the tags successfully, 
```suggestion
  def tags
    catch_aws_errors do
      tag_list = @aws.kms_client.list_resource_tags(key_id: @display_name).tags
      kms_tags(tag_list)
    end
  end
```

Reply:
fixed